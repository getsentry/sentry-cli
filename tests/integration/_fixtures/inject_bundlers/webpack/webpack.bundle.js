(()=>{"use strict";(function(n,o){!function(n,o){n(o)}(n,"boop")})((function(n){throw new Error(n)}))})();
//# sourceMappingURL=webpack.bundle.js.map