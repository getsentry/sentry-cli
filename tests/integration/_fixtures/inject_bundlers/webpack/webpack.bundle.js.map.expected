{"version":3,"file":"webpack.bundle.js","sources":["webpack://sourcemaps-playground/./src/foo.js","webpack://sourcemaps-playground/./src/bar.js","webpack://sourcemaps-playground/./src/index.js"],"sourceRoot":"","sourcesContent":["import { bar } from './bar.js';\n\nexport function foo(fn, msg) {\n  bar(fn, msg);\n}\n","export function bar(fn, msg) {\n  fn(msg);\n}\n","import { foo } from './foo.js';\n\nfunction wat(fn, msg) {\n  foo(fn, msg);\n}\n\nwat(function hello(msg) {\n  throw new Error(msg);\n}, 'boop');\n"],"names":["fn","msg","bar","foo","Error"],"mappings":";;oBAEO,SAAaA,EAAIC,ICFjB,SAAaD,EAAIC,GACtBD,EAAGC,EACL,CDCEC,CAAIF,EEKH,OFJH,EEDEG,EAGE,SAAeF,GACjB,MAAM,IAAIG,MAAMH,EAClB,G","debug_id":"397f1037-b958-54cc-bc62-267d5b7aff25"}